Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Update.hs

```diff
@@ -59,3 +78,51 @@ handler req = do
   logTagInfo "Payment completed : " ("bookingId " <> req.bookingId.getId <> ", rideId " <> req.rideId.getId)
 
 -- TODO store to DB
+
+handler EditLocationReq {..} = do
+  ride <- runInReplica $ QRide.findById rideId >>= fromMaybeM (RideNotFound rideId.getId)
+  person <- runInReplica $ QPerson.findById ride.driverId >>= fromMaybeM (PersonNotFound ride.driverId.getId)
+  whenJust origin $ \pickup -> do
+    startLocation <- buildLocation pickup
+    QL.create startLocation
+    pickupMapForBooking <- SLM.buildPickUpLocationMapping startLocation.id bookingId.getId DLM.BOOKING (Just person.merchantId) (Just person.merchantOperatingCityId)
```

Comment:
Why are we changing the booking? In this case on ride table will be changed, right?